### PR TITLE
Completing vi-former-members.md (#1657)

### LINKED ISSUE #1657: fix vi-former-members.md
- some items are not in chronological order
- end dates are missing for some VIs, look them up in their GitHub account history?

### PR DESCRIPTION
**Pull request by aakarnik**
### Description
I have completed the leave dates for the former members' list. Some of the dates do not look accurate - I have highlighted them in bold lettering. One member was not found on GitHub.

Once the dates are good, I can sort them in chronological order either by joining dates or leaving dates.

### RawGit preview link
https://rawgit.com/aakarnik/aakarnik.github.io/Fix_former_members/#!pages/vi/vi-former-members.md

### CODE REVIEW

**Comment by aakarnik** [comment]
Glad I could help! :)

I actually don't know why the red dot is appearing; maybe they get introduced when I edit it in my local repository and push it to GitHub. Is there an issue with syncing the branches?

File: pages/vi/vi-former-members.md [hunk 1]
```diff
@@ -8,35 +8,35 @@ To view the list of current members, go to [active members page](vi-team.md).
...
-|[daisywatson](profiles/daisywatson.md)| 2017-07-12 |
-|[alexkarasik](profiles/alexkarasik.md)| 2017-07-17 |
-|[l0rd3141](profiles/l0rd3141.md)| 2017-08-01 |
-|[sahityaLMaruvada](profiles/sahityaLMaruvada.md)| 2017-08-14 |
-|[ajGingrich](profiles/ajGingrich.md)| 2017-09-11 |
-|[thesiege208](profiles/thesiege208.md)| 2017-10-16 |
-|[aas919](profiles/aas919.md)| 2017-10-23 |
+|[hikaruit15](profiles/hikaruit15.md)| 2016-11-17 | 2017-03-28
+|[hannahgdubin](profiles/hannahgdubin.md)| 2017-01-01 | 2017-03-31
+|[sarvanivadali](profiles/sarvanivadali.md)| 2017-02-23 | 2017-03-25
```
Comment (dogi) [inline pages/vi/vi-former-members.md]: longer (august)

File: pages/vi/vi-former-members.md [hunk 2]
```diff
@@ -8,35 +8,35 @@ To view the list of current members, go to [active members page](vi-team.md).
...
-|[prajwalajayaprakash](profiles/prajwalajayaprakash.md)| 2017-06-30 |
-|[daisywatson](profiles/daisywatson.md)| 2017-07-12 |
-|[alexkarasik](profiles/alexkarasik.md)| 2017-07-17 |
-|[l0rd3141](profiles/l0rd3141.md)| 2017-08-01 |
-|[sahityaLMaruvada](profiles/sahityaLMaruvada.md)| 2017-08-14 |
-|[ajGingrich](profiles/ajGingrich.md)| 2017-09-11 |
-|[thesiege208](profiles/thesiege208.md)| 2017-10-16 |
-|[aas919](profiles/aas919.md)| 2017-10-23 |
+|[hikaruit15](profiles/hikaruit15.md)| 2016-11-17 | 2017-03-28
+|[hannahgdubin](profiles/hannahgdubin.md)| 2017-01-01 | 2017-03-31
```
Comment (dogi) [inline pages/vi/vi-former-members.md]: left when started (january)

File: pages/vi/vi-former-members.md [hunk 3]
```diff
@@ -8,35 +8,35 @@ To view the list of current members, go to [active members page](vi-team.md).
...
+|[sarvanivadali](profiles/sarvanivadali.md)| 2017-02-23 | 2017-03-25
+|[Chris-Boe](profiles/Chris-Boe.md)| 2017-02-26 | 2017-03-22
+|[blakedelee](profiles/BlakeDeLee.md)| 2017-03-22 | **2017-03-24**
+|[Loshma93](profiles/Loshma93.md) (**Wiki Intern Lead**)| 2017-03-31 | 2017-08-22
```
Comment (dogi) [inline pages/vi/vi-former-members.md]: stayed until January 2018

File: pages/vi/vi-former-members.md [hunk 4]
```diff
@@ -8,35 +8,35 @@ To view the list of current members, go to [active members page](vi-team.md).
...
+|[zeivhann](profiles/zeivhann.md) (**Mobile Intern Lead**)| 2017-04-01 | 2017-07-24
+|[Supriyaranjan](profiles/Supriyaranjan.md)| 2017-04-04 | 2017-05-17
+|[praneetharra](profiles/praneetharra.md)| 2017-04-21 | 2017-06-19
+|[shahswet](profiles/shahswet.md)| 2017-05-01 | 2017-05-03
+|[Yurockkk](profiles/Yurockkk.md)| 2017-05-05 | 2017-08-24
+|[sarah-lu102](profiles/sarah-lu102.md) (**Wiki Intern Lead**)| 2017-05-10 | 2017-07-31
+|[snazzybunny](profiles/snazzybunny.md) (**Embedded Systems Lead**)| 2017-05-11 | 2017-07-12
```
Comment (dogi) [inline pages/vi/vi-former-members.md]: left in November

File: pages/vi/vi-former-members.md [hunk 5]
```diff
@@ -8,35 +8,35 @@ To view the list of current members, go to [active members page](vi-team.md).
...
+|[duongdo27](profiles/duongdo.md)| 2017-05-12 | 2017-08-12
+|[aurinsomnia](profiles/aurinsomnia.md)| 2017-05-25 | 2017-08-02
```
Comment (dogi) [inline pages/vi/vi-former-members.md]: never started

File: pages/vi/vi-former-members.md [hunk 6]
```diff
@@ -8,35 +8,35 @@ To view the list of current members, go to [active members page](vi-team.md).
...
+|[NavneetKarnam](profiles/navneetkarnam.md)| 2017-06-26 | 2017-07-19
+|[tjyothsna16](profiles/tjyothsna16.md)| 2017-06-27 | 2017-07-07
+|[prajwalajayaprakash](profiles/prajwalajayaprakash.md)| 2017-06-30 | 2017-08-09
+|[daisywatson](profiles/daisywatson.md)| 2017-07-12 | 2017-09-20
+|[alexkarasik](profiles/alexkarasik.md)| 2017-07-17 | 2017-08-22
+|[l0rd3141](profiles/l0rd3141.md)| 2017-08-01 | **2017-08-01**
+|[sahityaLMaruvada](profiles/sahityaLMaruvada.md)| 2017-08-14 | **2017-08-17**
+|[ajGingrich](profiles/ajGingrich.md)| 2017-09-11 | 2017-11-07
+|[thesiege208](profiles/thesiege208.md)| 2017-10-16 | 2018-01-03 
+|[aas919](profiles/aas919.md)| 2017-10-23 |2018-02-10
```
Comment (dogi) [inline pages/vi/vi-former-members.md]: never really started

**Comment by xyb994** [comment]
Please wait for me to clean up some people who finished First steps and initial meeting but never contributed afterwards.

**Comment by aakarnik** [comment]
@xyb994  Okay. Let me know if I can contribute in anyway :+1:

**Pull request merged**